snyk/managers.py

```diff
@@ -255,10 +255,15 @@ def create(self, issue_id: str, fields: Any) -> Dict[str, str]:
         post_body = {"fields": fields}
         resp = self.client.post(path, post_body)
         response_data = resp.json()
-        if "jiraIssue" in response_data:
-            return response_data["jiraIssue"]
-        else:
-            raise SnykError
+        # The response we get is not following the schema as specified by the api
```

Comment:
Have you opened an issue against the API for this? Do we know if it's a bug in the API or the docs?